core/stdlib/std.ncl

```diff
@@ -3298,15 +3298,19 @@
             or a record. If a string is provided, it will be parsed into a record (in
             the `std.package.structured.PackageId` format).
 
-            Valid strings are of the format `github/<owner>/<repo>`, where `<owner>`
-            and `<repo>` are arbitrary strings that don't contain slashes.
+            Valid strings are of the format `github:<owner>/<repo>` or
+            `github:<owner>/<repo>/<path>`, where `<owner>` and `<repo>` are
```

Comment:
I think the documentation is wrong, since  `owner` and `repo` are actually pretty constrained, and not arbitrary strings.